app/queries/Connect.rb

```diff
@@ -0,0 +1,39 @@
+class Connect
+  def initialize(user1, user2)
+    @user1 = user1
+    @user2 = user2
+  end
+
+  def chat
+    @chatroom = Chatroom.create
+    [@user1, @user2].each do |user|
+      ChatroomsUser.create(user: user, chatroom: @chatroom)

```

Comment:
ChatroomsUser is unnecessary, that class needs to be deleted, and you also do not need to access it here